setup.cfg

```diff
@@ -0,0 +1,42 @@
+[bumpversion]
+current_version = 0.4.0
+commit = True
+tag = True
+
+[bumpversion:file:setup.py]
+search = version='{current_version}'
+replace = version='{new_version}'
+
+[bumpversion:file:eggshell/__init__.py]
+search = __version__ = '{current_version}'
+replace = __version__ = '{new_version}'
+
+[bdist_wheel]
+universal = 1
+
+[flake8]
+ignore = F401,E402
+max-line-length = 120
+exclude =
+	.git,
+	__pycache__,
+  .eggs/
+	docs/source/conf.py,
+	build,
+	dist,
+	src,
+	eggshell/legacy,
+
+#[aliases]
+# Define setup.py command aliases here
+#test = pytest
+
+[tool:pytest]
+addopts =
+	--strict
+	--tb=native
+	tests/
```

Comment:
Is that necessary ? It's inconvenient when you want to run a single test file, because by default it will run the entire tests directory, then run again the test file you're interested in.

Reply:
Right ... didn't notice that running a single test didn't work. Updated.